about.qmd

```diff
@@ -16,10 +16,14 @@ about:
 
 ---
 
-This repo is here to accelerate analysis of CosMx datasets. It contains code and writeups addressing challenges we've faced in analyzing CosMx data. After dozens of analyses, we have developed a sense of what works and what doesn't, and we have developed (sometimes partial) solutions for many frequent problems.
+This repo is here to accelerate analysis of CosMx datasets. It contains code and
+writeups addressing challenges we've faced in analyzing CosMx data. After dozens
+of analyses, we have developed a sense of what works and what doesn't, and we 
+have developed (sometimes partial) solutions for many frequent problems.
 
-
-> Note: _Code in this repo is provided without warranty or promise of support. This is not official NanoSting software and has not been subjected to the usual rigorous review and testing._
+> Note: _Code in this repo is provided without warranty or promise of support. 
+This is not official NanoSting software and has not been subjected to the usual 
```

Comment:
sp. NanoString

Reply:
Yes, I noticed that too. Prajan and I just finished refactoring that section. 

4e9c646e1090e1206fc8a92c024dd81cf66d5a5b